File: SourceCode/C/day045.c
/* Author: Ahmed A. M. ALOMAR
Date: February 14 - 2020
Description:
This program is an example of Iteration through Array elements  */


#include <stdio.h>

int main()
{
  int box[5];

  box[0] = 2;
  box[1] = 4;
  box[2] = 6;
  box[3] = 8;
  box[4] = 10;

  for (int i = 0; i < 5; i++)
  {
    printf("%d \n", box[i]);
  }
  
}

File: SourceCode/Python/day045.py
# ------------------------------------ #
# Author: Ahmed A. M. ALOMAR
# Date: May 25 - 2020
#
# Description:
# * Example of Creating Class Attributes
# ------------------------------------ #


class Plants():
  # Adding an attribute name to the init method
  def __init__(self, size):
    self.size = size


palm_tree = Plants(10)
flower = Plants(1)

print(palm_tree.size)
print(flower.size)
